examples/src/fibonacci/utils.rs

```diff
@@ -17,6 +20,18 @@ pub fn compute_fib_term(n: usize) -> BaseElement {
     t1
 }
 
+pub fn compute_fib_term_f64(n: usize) -> F64 {
+    let mut t0 = F64::ONE;
+    let mut t1 = F64::ONE;
+
+    for _ in 0..(n - 1) {
+        t1 = t0 + t1;
+        core::mem::swap(&mut t0, &mut t1);
+    }
+
+    t1
+}
```

Comment:
Instead of creating a new function, we can probably make `compute_fib_term()` function generic.

Reply:
done